client/src/components/ScrollMenu.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import styled from "styled-components/macro";
+import Test from "../assets/test.jpg";
+
+const Div = styled.div`
```

Comment:
Naming, also try finding your own naming conventions, sometimes you name the outer Container `Div` sometimes `Container`